src/engine/renderer.js

```diff
@@ -30,6 +33,40 @@ export const setRenderer = () => {
   rttB = new THREE.WebGLRenderTarget(null, null, renderTargetParameters)
 
   quadScene = new QuadScene(rttA, rttB)
+  quadMixUniform = quadScene.material.uniforms.mixRatio
+
+  composer = new EffectComposer(renderer)
+}
+
+export const setPostProcessing = () => {
+  const state = store.getState()
+  const stateScenes = getScenes(state)
+
+  composer.reset()
+
+  mainPass = new RenderPass(quadScene.scene, quadScene.camera)
+  mainPass.renderToScreen = true
+  composer.addPass(mainPass)
+
+  const passes = []
+
+  stateScenes.forEach(scene => {
+    scene.sketchIds.forEach(sketchId => {
+      const module = engine.sketches[sketchId]
+      // Do any postprocessing related setup for this sketch
+      if (scene.settings.globalPostProcessingEnabled && module.initiatePostProcessing) {
```

Comment:
curious to know what your thoughts are on having effect sketches extend a base class with a flag so we can detect it is an effect (type checking is typescript only right? little rusty) and possibly some utility functions/params?

Reply:
Hmm what is the advantage of doing that? Having the "global post processing" setting is more of a future proofing thing. It needs to be checked at the moment because we only have global post processing (where the effect works on top of any crossfader stuff). Later on we'll have a scene specific post processing (so will only affect the channel as opposed to the whole fade mix) .